lib/publiccloud/ssh_interactive.pm

```diff
@@ -18,7 +18,16 @@ our @ISA = qw(Exporter);
 our @EXPORT = qw(ssh_interactive_tunnel ssh_interactive_leave);
 
 sub ssh_interactive_tunnel {
-    my ($instance) = @_;
+    # Establish the ssh interarctive tunnel to the publiccloud instance.
+    # Optional arguments: 'force => 1' - reestablish tunnel, also if already established.
+
+    my $instance = shift;
+    my %args = testapi::compat_args({cmd => undef}, ['cmd'], @_);
+    $args{force} //= 0;
+
+    # $serialdev should be always set from os-autoinst/testapi.pm
+    die '$serialdev is not set' if (!defined $serialdev || $serialdev eq '');
+    return if ($args{force} != 1 && get_var('_SSH_TUNNELS_INITIALIZED', 0) == 1);
```

Comment:
```suggestion
    return if ($args{force} != 1 && check_var('_SSH_TUNNELS_INITIALIZED', 1));
```

Reply:
Apart from style, is there any other reason for doing this change? I honestly don't like `check_var` and try to avoid that whenever possible.

Reply:
Yes, this is of course minor.